content/300-guides/200-deployment/110-deployment-guides/400-deploying-to-aws-lambda.mdx

```diff
@@ -344,6 +344,27 @@ This ensures the packaged archive is as small as possible.
 
 But, if you are using `serverless-webpack`, you cannot use that method.
 
+### Lambdas with arm64 architectures
+
+Lambda functions that use [arm64 architectures (AWS Graviton2 processor)](https://docs.aws.amazon.com/lambda/latest/dg/foundation-arch.html#foundation-arch-adv) should use an `arm64` precompiled binary.
```

Comment:
```suggestion
Lambda functions that use [arm64 architectures (AWS Graviton2 processor)](https://docs.aws.amazon.com/lambda/latest/dg/foundation-arch.html#foundation-arch-adv) must use an `arm64` precompiled binary.
```

We avoid "should" in docs, because it's ambiguous (it can mean "ideally you would do this" or "it might happen").